docs/configure/styling-and-css.md

```diff
@@ -22,66 +32,136 @@ Storybook is pre-configured to recognize imports for CSS files. To add global CS
 />
 
 <!-- prettier-ignore-end -->
+If your component files import their CSS files, this will work too. However, if you're using CSS processor tools like Sass or Postcss, you may need some more configuration.
 
-If your component files import their CSS files, this will work too. The noticeable exception to this is if you're using CSS processor tools like Sass or Postcss.
+## CSS modules
 
-## CSS processors
+### Vite
+Vite comes with CSS modules support out-of-the-box. If you have customized the CSS modules configuration in your `vite.config.js` this will automatically be applied to your Storybook as well. Read more about [Vite's CSS modules support](https://vitejs.dev/guide/features.html#css-modules).
 
-If you're using Vite as your builder, you're covered! Vite supports Sass and PostCSS out-of-the-box 🎉
+### Webpack
+<IfRenderer renderer='react'>
+<Callout
+  variant="info"
+  icon="📣"
+  title="Using `@storybook/nextjs`?"
+>
 
-However, if you're using Webpack and want to use Sass and PostCss, you'll need some extra configuration. We recommend installing [`@storybook/addon-styling-webpack`](https://storybook.js.org/addons/@storybook/addon-styling-webpack/) to help you configure these tools. Or if you'd prefer, you can customize [Storybook's webpack configuration yourself](../builders/webpack.md#override-the-default-configuration) to include the appropriate loader(s).
+Storybook recreates your Next.js configuration, so you can use CSS modules in your stories without any extra configuration.
 
-## CSS-in-JS
+</Callout>
+</IfRenderer>
```

Comment:
This will result in an empty **Webpack** heading for any renderer that isn't `react`.